man/man8/zfsprops.8

```diff
@@ -876,14 +889,16 @@ is equivalent to
 Faster speeds at the cost of the compression ratio can be requested by
 setting a negative
 .Sy zstd
-level.  This is done using
+level.
+This is done using
 .Sy zstd-fast- Ns Em N ,
 where
 .Em N
 is an integer in [1-9,10,20,30,...,100,500,1000] which maps to a negative
 .Sy zstd
-level. The lower the level the faster the compression - 1000 provides
-the fastest compression and lowest compression ratio.
+level.
+The lower the level the faster the compression -
```

Comment:
nit: instead of a dash, use a colon (and add a comma). "The lower the level, the faster the compression: 1000 provides the fast compression and lowest compression ratio."

Reply:
yeah